packages/node/src/methods/errors.ts

```diff
@@ -7,5 +7,9 @@ export const ERRORS = {
   NO_PROPOSED_APP_INSTANCE_FOR_APP_INSTANCE_ID:
     "No proposed AppInstance exists for the given appInstanceId",
   NULL_INITIAL_STATE_FOR_PROPOSAL:
-    "A proposed AppInstance cannot have an empty initial state"
+    "A proposed AppInstance cannot have an empty initial state",
+  NO_APP_INSTANCE_FOR_TAKE_ACTION:
+    "No AppInstanceId specified to takeAction on",
+  NO_APP_CONTRACT_ADDR: "The App Contract address is empty",
+  INVALID_ACTION: "Invalid action taken"
```

Comment:
You could include the error from the blockchain here. Usually it will actually revert with a message like: "cannot place X here!"

Reply:
It does - that's how the error sanitation behaves right now. So are you suggesting we make this a function instead of a constant?